### PR TITLE
New index to search tree path

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
Search will accept new index for tree structure description

https://oat-sa.atlassian.net/browse/CON-23

### CODE REVIEW

File: src/QueryBuilder.php [hunk 1]
```diff
@@ -34,6 +34,7 @@ class QueryBuilder extends ConfigurableService
 
     private const STANDARD_FIELDS = [
         'class',
+        'parents',
```
Comment (gabrielfs7) [inline src/QueryBuilder.php:37]: Can you please put a @TODO to avoid us to forward elastic search syntax from FE directly?

**Review by gabrielfs7 — APPROVED** [review]
Good job! 

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by shpran — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

File: src/QueryBuilder.php (hunk 1 above)
Comment (bartlomiejmarszal) [reply src/QueryBuilder.php]: here? I think we should add this to frontend

**Pull request merged**